README.md

```diff
@@ -942,18 +942,21 @@ able to use tools from ```scripts/``` anywhere.
 * `prj.conf` - The main configuration file for a Zephyr application; overrides
 settings from a defconfig file in the Zephyr tree. In the ZJS builds, we
 assemble the prj.conf file at build time from other fragments.
-* `prj.mdef` - Another configuration file for a Zephyr application; we use it to
-configure the heap size available to the ZJS API.
 
 ### Subdirectories
 - `arc/` - Contains sensor subsystem code for ARC side of the Arduino 101.
+- `cmake/` - Contains CMake build files for building Zephyr and JerryScript
 - `deps/` - Contains dependency repos and scripts for working with them.
 - `docs/` - Documentation in Markdown format (use API.md as index).
+- `fragments/` - Contains project file fragments for project configuration.
+- `modules/` - Contains different Javascript modules that can be loaded.
+- `node_modules/` - Contains different Javascript modules that can be loaded.
```

Comment:
This was new to me... and still is. :) I don't seem to have it... is it just on your system?

Reply:
sorry, maybe it's only on my system, i'll remove it.